docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -2138,13 +2142,58 @@ public class CustomGlobalFilter implements GlobalFilter, Ordered {
 ----
 ====
 
+=== The Gateway Metrics Filter
+
+To enable gateway metrics, add spring-boot-starter-actuator as a project dependency. Then, by default, the gateway metrics filter runs as long as the property `spring.cloud.gateway.metrics.enabled` is not set to `false`. This filter adds a timer metric named `spring.cloud.gateway.requests` with the following tags:
```

Comment:
Change to:

To enable gateway metrics, add `spring-boot-starter-actuator` as a project dependency. Then, by default, the gateway metrics filter runs as long as the `spring.cloud.gateway.metrics.enabled` property is not set to `false`. This filter adds a timer metric named `spring.cloud.gateway.requests` with the following tags:

(I would put each sentence on its own line, too. That helps with future maintenance.)